src/pudl/metadata/fields.py

```diff
@@ -3388,6 +3426,10 @@
         "description": "Indicates whether the plant is regulated or non-regulated.",
     },
     "report_date": {"type": "date", "description": "Date reported."},
+    "report_date_plant_gen": {
+        "type": "date",
+        "description": "Associated plant_gen's date reported.",
+    },
```

Comment:
I feel like report date doesn't need the `plant_gen` suffix?

Reply:
your not wrong here... both the `report_date` column and the `plant_id_eia` are always consistent across ppe records within a given plant/year. I personally lean towards making all of the columns consistent and clear in that they are columns from the plant_gen record, even though this is a little extraneous. I mostly just think being inconsistent about the suffix will be harder to explain.